### DISCUSSION TITLE
Would it be possible to add a "stop" button?

### DISCUSSION THREAD

**Discussion by GalaxyTimeMachine** [hlky/stable-diffusion#145] [Ideas]
Sometimes I'll start running a multiple sized batch and realise there was a typo, or I'd missed something, or the initial results are wrong. It would be nice if there was a stop/panic button that interrupted the run, instead of waiting until it has finished and then deleting them all and starting again.

**Comment by artificialguybr** [comment]
A stop button would really be great.

**Reply by OWKenobi** [reply]
Also, the stop button would be great, so I could start a batch of, lets say, 1000 pictures, and after 20 minutes figure out I don't want to wait any longer, press the stop button, and see what has been generated so far.

I tried Control+C in the console (which should be pretty easy to monitor), but unfortunately, it quits the whole application.

**Comment by F-302** [comment]
Yes, that makes sense. Especially if you run the WebUI remotely and don't have the ability or desire to constantly run to a PC.

**Comment by GalaxyTimeMachine** [comment]
Stop and refresh buttons have been added!! Awesome, thank you! 😊
